helix-core/src/test/java/org/apache/helix/integration/manager/TestParticipantManager.java

```diff
@@ -101,8 +103,13 @@ public void simpleIntegrationTest() throws Exception {
         "MasterSlave", true); // do rebalance
 
     String instanceName = "localhost_12918";
-    HelixManager participant =
-        new ZKHelixManager(_clusterName, instanceName, InstanceType.PARTICIPANT, ZK_ADDR);
+    int resetTimeout = 300;
```

Comment:
This is very good question, it's not new logic, and I don't see a maximum value from https://github.com/apache/helix/pull/1920
I assume the idea is to delegate this to user for a reasonable value, it's hard to set a global max and override. Let me know if you think otherwise. Thanks.